tests/queries/0_stateless/03403_truncate_all_tables_like.sql

```diff
@@ -0,0 +1,78 @@
+CREATE DATABASE IF NOT EXISTS d;
```

Comment:
```suggestion
CREATE DATABASE IF NOT EXISTS  {CLICKHOUSE_DATABASE:Identifier};
```
Might mostly fix the fast test [failures](https://s3.amazonaws.com/clickhouse-test-reports/json.html?PR=78597&sha=31c7ebcbccf1bd858ea20da3f5fcb63d701cf3bf&name_0=PR&name_1=Fast+test&name_2=Tests) seen on CI currently.